### ISSUE TITLE
French translation issue

### ISSUE THREAD

**Issue by ferllings** [ThePacielloGroup/CCAe#149]
Oubli du s final : Analyseur de contraste des couleurs

Remplacer | par
-- | --
Fail | Échec
Pass | Conforme
Démonstration | Exemple
(Obtenu par Édition-Copier les résultats)Foreground: #FE0000Background: #D9D9D9The contrast ratio is: 2.9:11.4.3 Contrast (Minimum) (AA)    Fail for large and regular text1.4.6 Contrast (Enhanced) (AAA)    Fail for large and regular text1.4.11 Non-text Contrast (AA)    Fail for UI components and graphical objects | (Obtenu par Édition-Copier les résultats)Premier plan : #FE0000Arrière-plan : #D9D9D9Le ratio de contraste est : 2.9:11.4.3 Contraste (minimum) (AA)Échec pour texte normal et grand texte1.4.6 Contrase (augmenté) (AAA)Échec pour texte normal et grand texte1.4.11 Non-text Contrast (AA)Échec pour composants UI et objets graphiques
